plugin/nfs/nfs_clib.c

```diff
@@ -0,0 +1,166 @@
+#include <Python.h>
+#include <stdint.h>
+#include <stdbool.h>
+#include <sys/statvfs.h>
+#include <mntent.h>
+
+#include <libstoragemgmt/libstoragemgmt.h>
+
+
+#define _BUFF_SIZE   1024
+#define LSF_LOCAL_MOUNTS "/proc/self/mounts"
+
+
+
+#if PY_MAJOR_VERSION > 2
+#define PyInt_FromLong PyLong_FromLong
+#endif
+
+#if PY_MAJOR_VERSION >= 3
+    #define MOD_DEF(name, methods) \
+        static struct PyModuleDef moduledef = { \
+            PyModuleDef_HEAD_INIT, name, NULL, -1, methods, NULL, NULL, NULL, \
+            NULL}; \
+        return PyModule_Create(&moduledef);
+#else
+    #define MOD_DEF(name, methods) \
+        Py_InitModule(name, methods);
+#endif
+
+#define _UNUSED(x) (void)(x)
+
+
+static PyObject * find_fsid_bypath(PyObject * self, PyObject *args)
```

Comment:
Please be consistent on using `void *name` or `void * name` code style.
Using `void *name` code style is suggested.